src/racer/codecleaner.rs

```diff
@@ -1,11 +1,20 @@
 use core::{Point, SourceByteRange};
 
+/// Type of the string
+#[derive(Clone, Copy, Debug)]
+enum StringType {
+    /// Raw(n) => raw string started with n #s
+    Raw(usize),
+    /// normal string starts with "
+    NotRaw,
```

Comment:
Please use `Cooked` to match [`syn`'s convention](https://docs.rs/syn/0.12.10/syn/enum.StrStyle.html)